kafka_consumer/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # CHANGELOG - kafka_consumer
 
+## 1.9.1 / 2019-06-01
+
+* [Fixed] Fix style. See [#3838](https://github.com/DataDog/integrations-core/pull/3838).
+* [Fixed] Handle empty topics and partitions. See [#3807](https://github.com/DataDog/integrations-core/pull/3807).
+* [Fixed] Add kakfa to the description. See [#3802](https://github.com/DataDog/integrations-core/pull/3802).
```

Comment:
This is a change to a comment in the yaml file. I think we should revisit whether this needs a changelog entry (or update the wording to reflect what exactly this is doing :) )